### PR TITLE
Iow 857 - Add a table of groundwater field visit data

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [X] Update the changelog appropriately

Title
-----------
Add a table of groundwater field visit data

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/src/scripts/monitoring-location/components/hydrograph/data-table.js [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
 
-const CONTAINER_ID = 'iv-data-table-container';
+const VALUE_NAMES = {
+    iv: ['parameterName', 'dateTime', 'result', 'approvals', 'masks'],
+    gw: ['parameterName', 'dateTime', 'result']
+};
 
-const drawTableBody = function(table, ivData) {
-    table.select('tbody').remove();
+const CONTAINER_ID = {
```
Comment (abriggs-usgs) [inline assets/src/scripts/monitoring-location/components/hydrograph/data-table.js:25]: Nice clean way of setting up multiple tables. 👍

**Review by abriggs-usgs — APPROVED** [review]
Looks good!

File: assets/src/scripts/monitoring-location/components/hydrograph/download-links.test.js [hunk 2]
```diff
@@ -475,7 +475,7 @@ describe('monitoring-location/components/hydrograph/download-links', () => {
             });
         });
 
-        fit('Renders the correct links when only groundwater data is available', () => {
+        it('Renders the correct links when only groundwater data is available', () => {
```
[the remark this answers is not in the document]
Comment (mbucknell) [reply assets/src/scripts/monitoring-location/components/hydrograph/download-links.test.js:478]: I am surprised the Codacy didn't ping it. It's dinged on my IDE.

File: wdfn-server/waterdata/templates/macros/components.html [hunk 3]
```diff
@@ -52,7 +52,7 @@ <h2 class="usa-accordion__heading">
                 <button class="usa-accordion__button"
                         aria-expanded="false" aria-controls="iv-data-table-container"
                         ga-on="click" ga-event-category="accordion" ga-event-action="interactionWithHydrographDataTableAccordion">
-                    Table of hydrograph data
+                    Table(s) of hydrograph data
```
[the remark this answers is not in the document]
Comment (mbucknell) [reply wdfn-server/waterdata/templates/macros/components.html]: I like your suggestion. I am going to change it.

**Pull request merged**